### PR TITLE
Bug 1817466 - [pushapkscript] Add new channel/certificate_alias names

### PR DESCRIPTION
**Pull request by gbrownmozilla**
Add new channel and certificate_alias names. Once the new names are in use, a subsequent PR will remove the old configuration.

### CODE REVIEW

**Comment by gbrownmozilla** [comment]
new fenix_nightly.pem == existing fenix_production.pem
new fenix_beta.pem == existing release.pem
new fenix_release.pem == existing release.pem

File: pushapkscript/docker.d/init_worker.sh [hunk 1]
```diff
@@ -78,11 +78,16 @@ case $COT_PRODUCT in
         echo $GOOGLE_CREDENTIALS_FENIX_BETA | base64 -d >        $GOOGLE_CREDENTIALS_FENIX_BETA_PATH
         echo $GOOGLE_CREDENTIALS_FENIX_RELEASE | base64 -d >     $GOOGLE_CREDENTIALS_FENIX_RELEASE_PATH
 
+        # TODO bug 1817466 - remove
```
Comment (JohanLorenzo) [inline pushapkscript/docker.d/init_worker.sh:81]: 👍

File: pushapkscript/docker.d/worker.yml [hunk 2]
```diff
@@ -20,7 +20,15 @@ products:
           skip_check_same_locales: true
           skip_checks_fennec: true
           apps:
-            # TODO: Rename this product `beta`
+            fenix-beta:
+              package_names: [ "org.mozilla.firefox_beta" ]
+              certificate_alias: 'fenix-beta'
+              google:
+                # TODO Revisit the default track once Fenix is kicked off by shipit
```
Comment (JohanLorenzo) [inline pushapkscript/docker.d/worker.yml]: 🤔 I know this comment is just a copy-over, but I don't remember what it means 😅 For sure, Fenix has been kicked off by shipit for more than 2 years. As far as I know, there's noneed to change anything related to the Google Play tracks.

Thus, I'd recommend to remove this comment (and the same one down below) in order not to confuse future readers.

File: pushapkscript/docker.d/worker.yml [hunk 3]
```diff
@@ -100,41 +139,68 @@ products:
           skip_check_same_locales: true
           skip_checks_fennec: true
           apps:
+            fenix-nightly:
+              package_names: [ "org.mozilla.fenix" ]
+              certificate_alias: 'fenix'
```
Comment (JohanLorenzo) [inline pushapkscript/docker.d/worker.yml:140]: Nit: I wonder if we should call this certificate `fenix-dep`. Having `fenix`, `fenix-release` and all looks confusing. `fenix-dep` would clarify the intent behind this certificate, in my opinion.

File: pushapkscript/docker.d/worker.yml [hunk 4]
```diff
@@ -163,13 +229,29 @@ products:
                 default_track: 'alpha'
                 service_account: 'dummy'
                 credentials_file: { "$eval": "GOOGLE_CREDENTIALS_FOCUS_DEP_PATH" }
+            focus-nightly:
+              package_names: ["org.mozilla.focus.nightly"]
+              certificate_alias: 'focus'
```
Comment (JohanLorenzo) [inline pushapkscript/docker.d/worker.yml:230]: Nit: Same comment about `focus` vs `focus-dep`.

File: pushapkscript/files/fenix_beta.pem [hunk 5]
```diff
@@ -1,23 +1,22 @@
 -----BEGIN CERTIFICATE-----
-MIIDyTCCArGgAwIBAgIEeW86BDANBgkqhkiG9w0BAQsFADCBlDELMAkGA1UEBhMC
+MIIDpjCCAo6gAwIBAgIETHL9iDANBgkqhkiG9w0BAQUFADCBlDELMAkGA1UEBhMC
```
Comment (JohanLorenzo) [inline pushapkscript/files/fenix_beta.pem:2]: Okay to change these existing certificate. They are unused anyway 👌

**Review by JohanLorenzo — APPROVED** [review]
Looks good to me. Feel free to ignore the nit comments, your PR already makes the config much cleaner 👍

File: pushapkscript/docker.d/worker.yml (hunk 2 above)
Comment (gbrownmozilla) [reply pushapkscript/docker.d/worker.yml]: Thanks. Done

File: pushapkscript/docker.d/worker.yml (hunk 3 above)
Comment (gbrownmozilla) [reply pushapkscript/docker.d/worker.yml:140]: I hear you! But if I change fenix/focus, I would also want to change reference_browser, which I think has a slightly different configuration, which... I think it's not quite worth the extra effort -- leaving it as-is.

File: pushapkscript/files/fenix_beta.pem (hunk 5 above)
Comment (gbrownmozilla) [reply pushapkscript/files/fenix_beta.pem:2]: Thanks for the confirmation.

**Pull request merged**